ledfx/core.py

```
@@ -513,6 +553,21 @@ async def async_stop(self, exit_code):
             self.events.fire_event(LedFxShutdownEvent())
+
+            # Stop audio device monitor
+            if self.audio_device_monitor:
+                try:
+                    # Unregister the event listener
+                    if hasattr(self, "_remove_audio_device_listener"):
+                        self._remove_audio_device_listener()
+
+                    # Stop monitoring
+                    self.audio_device_monitor.stop_monitoring()
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)